CMakeLists.txt

```diff
@@ -454,11 +454,16 @@ message(STATUS "Architecture-specific source files: ${ZLIB_ARCH_SRCS}")
 #============================================================================
 
 macro(generate_cmakein input output)
-    execute_process(COMMAND sed "/#define ZCONF_H/ a\\\n#cmakedefine Z_HAVE_UNISTD_H\\\n#cmakedefine Z_HAVE_STDARG_H\n"
-                    INPUT_FILE ${input}
-                    OUTPUT_FILE ${output}
-)
-
+    file(REMOVE ${output})
+    file(STRINGS ${input} _lines)
```

Comment:
`file(STRINGS)` is available in CMake 2.8.4

https://cmake.org/cmake/help/v2.8.4/cmake.html#command:file